x-pack/plugins/maps/public/classes/sources/es_search_source/es_search_source.tsx

```diff
@@ -186,27 +225,45 @@ export class ESSearchSource extends AbstractESSource {
   }
 
   // Returns sort content for an Elasticsearch search body
-  _buildEsSort() {
+  _buildEsSort(): Array<Record<string, SortDirection | SortDirectionNumeric>> {
     const { sortField, sortOrder } = this._descriptor;
+
+    if (!sortField) {
+      throw new Error('Cannot build sort');
+    }
     return [
       {
         [sortField]: {
-          order: sortOrder,
+          order: (sortOrder as unknown) as SortDirection,
```

Comment:
sortOrder is typed as `maps/common/constants/SORT_ORDER`. Thanks for finding `SortDirection`. `SORT_ORDER` is an unneeded duplication of `SortDirection`. Can you replace `SORT_ORDER` with `SortDirection`?